integration/setup/setup.go

```diff
@@ -60,6 +60,7 @@ var (
 	logLevelF   = zap.LevelFlag("log-level", zap.DebugLevel, "log level for tests")
 
 	disablePushdownF = flag.Bool("disable-pushdown", false, "disable pushdown")
+	enableNewAuthF   = flag.Bool("enable-new-auth", false, "enable new auth")
```

Comment:
Let's remove that flag and always enable it in tests

Reply:
Fixed.